### PR TITLE
chore(deps): bump fiatconnect-sdk & fiatconnect-types

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

Bump to latest and fix breaking changes. https://linear.app/valora/issue/ACT-496/add-response-schema-validation-to-fiatconnect-sdk

### CODE REVIEW

File: src/fiatconnect/saga.ts [hunk 1]
```diff
@@ -150,12 +150,13 @@ export function* handleSubmitFiatAccount({
     quote.getProviderBaseUrl(),
     quote.getProviderApiKey()
   )
+
   const postFiatAccountResponse: Result<PostFiatAccountResponse, ResponseError> = yield call(
-    [fiatConnectClient, 'addFiatAccount'],
+    { context: fiatConnectClient, fn: fiatConnectClient.addFiatAccount },
```
Comment (satish-ravi) [inline src/fiatconnect/saga.ts:155]: is this change required?

**Review by satish-ravi — APPROVED** [review]

File: src/fiatconnect/saga.ts (hunk 1 above)
Comment (jh2oman) [reply src/fiatconnect/saga.ts:155]: no its not, just a different way of using `yield call`
